package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "private-tx",
+  "version": "0.1.0",
+  "description": "Companion contract for Parity's Private Transactions feature",
+
+  "author": "Parity Technologies <admin@parity.io>",
+  "homepage": "https://github.com/parity-contracts/private-tx",
+  "license": "Apache-2.0",
+
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/parity-contracts/private-tx.git"
+  },
+  "bugs": {
+    "url": "https://github.com/parity-contracts/private-tx/issues"
+  },
+
+  "scripts": {
+    "ci": "concurrently \"npm run solium\" \"npm run truffle-with-rpc\" \"npm run solidity-coverage\"",
+
+    "ganache": "ganache-cli -p 8547",
+    "solidity-coverage": "solidity-coverage",
+    "solium": "solium --dir contracts/",
+    "test": "npm run solium && npm run truffle-with-rpc",
+    "truffle": "truffle test",
+    "truffle-with-rpc": "concurrently --success first --kill-others \"npm run ganache\" \"npm run truffle\""
+  },
+
+  "devDependencies": {
+    "babel-polyfill": "^6.26.0",
```

Comment:
I didn't add babel in `auto-updater` because the js code will only be run on node, assuming a recent version of node it has all the features we need.